cli/src/apiToken.ts

```diff
@@ -2,16 +2,44 @@ import { getAccessTokenPath } from "./accessTokenPath.ts";
 
 const serverURL = Deno.env.get("RMMBR_SERVER");
 
-export const apiToken = () =>
-  getAccessToken()
-    .then(getOrCreateApiToken)
+type APITokenInterface =
+  | { create: true }
+  | { delete: string }
+  | { list: true }
+  | { get: true }
+  | Record<string, never>;
+
+export const apiToken = (
+  cmd: APITokenInterface,
+) => {
+  const action: (_: string) => Promise<unknown> = "create" in cmd
+    ? createApiToken
+    : "delete" in cmd
+    ? deleteApiToken(cmd["delete"])
+    : "list" in cmd
+    ? listApiTokens
+    : getOrCreateApiToken; // The default action is "get or create"
+
+  return getAccessToken()
+    .then(action)
     .then(console.log);
+};
 
-const apiTokenRequest = (accessToken: string, method: "GET" | "POST") =>
+const apiTokenRequest = (
+  accessToken: string,
```

Comment:
move this after the currying, so when you build `createApiToken` and `listApiTokens` you can avoid the duplication

e.g. 
```js
const listApiTokens = apiTokenRequest("GET", null); // this would allow you to make body non-optional, without incurring any reptition
const createApiTokens = apiTokenRequest("POST", {action: "create"});

```